addons/l10n_tw_chart/account_chart_type.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+<!--
+
+   # Copyright (C) 2014-now  Jason Wu (jaronemo@msn.com)
```

Comment:
please do not put copyright notice in each file, marking you as the author is enough